src/main/java/com/coveros/selenified/element/check/Contains.java

```diff
@@ -208,22 +208,22 @@ default String checkValue(String expectedValue, double waitFor, double timeTook)
      * @return String[]: all of the select options of the element. null will be returned if the element isn't present or a select
      */
     @SuppressWarnings("squid:S1168")
-    default String[] checkSelectOption(String expectedOption, double waitFor, double timeTook) {
+    String[] checkSelectOption(String expectedOption, double waitFor, double timeTook) {
         // record the action, and check for select
-        if (!isPresentSelect(getElement().prettyOutput() + " with the option <b>" + expectedOption +
+        if (!isPresentSelect(this.element.prettyOutput() + " with the option <b>" + expectedOption +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " with the option <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)